.gitignore

```diff
@@ -5,6 +5,8 @@ __pycache__/
 htmlcov/
 dist/
 workspace/
+pilot/
```

Comment:
Why? we have `pilot/` in the new codebase (need it for the import to go smoothly, although we'll get rid of it).